Blutalkohol.java

```diff
@@ -0,0 +1,76 @@
+package Blutalkohol;
+
+import java.util.Scanner;
+public class Blutalkohol {
+
+	public static double rundenZweiteNachkommastelle(double zahl)
+		
+		{
```

Comment:
closing brackets at the end of the function
public static double ...(double zahl) {